core/scheduler/src/main/kotlin/Repeating.kt

```diff
@@ -0,0 +1,27 @@
+import java.time.DayOfWeek
+
+sealed class Repeating {
+    abstract val hour: Int
+    abstract val minute: Int
+    abstract val second: Int
+
+    data class Daily(
+        override val hour: Int = 0,
+        override val minute: Int = 0,
+        override val second: Int = 0
+    ) : Repeating()
```

Comment:
I like that kotlinx-datetime doesn't tie us to JVM here, so I don't see a benefit to switching

Reply:
Yes that's okay, doesn't kotlinx-datetime have a Local time / OffsetTime or something like that? What I meant was "let's use that instead of hour/minute/second parameters

Reply:
Oh I see now, I think it does so we could use that